### ISSUE TITLE
Add options to skip getDepotLocation and changesOnDepot

### ISSUE THREAD

**Issue by JayBazuzi** [Zougi/posh-p4#5]
These can be slow, especially for very large depots.

**Comment by fredericrous** [comment]
Yes it is slow, but removing these two options would have the same effect as not including the prompt, would it not?
I could add a note in the readme regarding the performance issue.

The solution I see to solve this performance issue would be to make a C# dll especially dedicated to check depot location and changes in a background task...and of course signal in the prompt when data are still waiting info back from the dll..
I never took the time to make this improvement, and I do not use p4 on a daily basis like I used to.
I would recommend to use the posh-p4 package not for its prompt but for the p4 autocompletion

EDIT: I just saw you made PRs I will review them =) thank you

**Comment by JayBazuzi** [comment]
Thanks. 

While getDepotLocation and changesOnDepot are slow, pendingStatus is fast and still useful.

I'll look in to sending a PR to address this.
